Simplified/Reader2/UI/NYPLBaseReaderViewController.swift

```diff
@@ -252,10 +252,6 @@ class NYPLBaseReaderViewController: UIViewController, NYPLBackgroundWorkOwner, L
     guard let bookmark = bookmarksBusinessLogic.addBookmark(location) else {
       let alert = NYPLAlertUtils.alert(title: "Bookmarking Error",
                                        message: "A bookmark could not be created on the current page.")
-      let action = UIAlertAction(title: NSLocalizedString("OK", comment: ""),
```

Comment:
oh is this just unneeded since it's the default?

Reply:
Yeah, `NYPLAlertUtils` has a default `Ok` button